### PR TITLE
new Creating packages getting started

### PR DESCRIPTION
**Pull request by memsharded**
Docs for https://github.com/conan-io/conan/pull/9516

### CODE REVIEW

File: creating_packages/getting_started.rst [hunk 1]
```diff
@@ -1,377 +1,221 @@
...
-                                  '''PROJECT(MyHello)
-    include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-    conan_basic_setup()''')
+        options = {"shared": [True, False], "fPIC": [True, False]}
+        default_options = {"shared": False, "fPIC": True}
+
+        # Sources are located in the same place as this recipe, copy them to the recipe
+        exports_sources = "src/*"
+
+        def config_options(self):
```
Comment (lasote) [inline creating_packages/getting_started.rst:60]: Too much for starting? Maybe remove the `fPIC` option? and explain the options later?

File: creating_packages/getting_started.rst [hunk 2]
```diff
@@ -1,377 +1,221 @@
...
-- The ``settings`` field defines the configuration of the different binary packages. In
-  this example, we defined that any change to the OS, compiler, architecture or build type will
-  generate a different binary package. Please note that Conan generates different binary packages for
-  different introduced configuration (in this case settings) for the same recipe.
 
-  Note that the platform on which the recipe is running and the package being built differ from
-  the final platform where the code will be running (``self.settings.os`` and ``self.settings.arch``) if
-  the package is being cross-built. So if you want to apply a different build depending on the current
-  build machine, you need to check it:
+Without going into detail, these are the basics:
```
Comment (lasote) [inline creating_packages/getting_started.rst:84]: If I read this I would say... wow, luckily it is only the basics because what a s** load of text!

**Review by lasote — APPROVED** [review]

File: creating_packages/getting_started.rst [hunk 3]
```diff
@@ -1,377 +1,221 @@
...
+    File saved: src/hello.cpp
+    File saved: src/hello.h
+    File saved: test_package/conanfile.py
+    File saved: test_package/src/CMakeLists.txt
+    File saved: test_package/src/example.cpp
 
-This will generate the following files:
 
-.. code-block:: text
+On the root folder, there is a *conanfile.py* which is the main recipe file, responsible for
```
[the remark this answers is not in the document]
Comment (memsharded) [reply creating_packages/getting_started.rst:34]: Done

File: creating_packages/getting_started.rst (hunk 1 above)
Comment (memsharded) [reply creating_packages/getting_started.rst:60]: This is the recipe exactly as it is created with ``conan new --template=cmake_lib``, not sure if we should hide it from the docs. I think the real feature is to automate that in Conan, so it doesn't need to be in every recipe, but at the moment, not sure what is the best approach.

File: creating_packages/getting_started.rst (hunk 2 above)
Comment (memsharded) [reply creating_packages/getting_started.rst:84]: Think it is better to try to reduce it?

Comment (lasote) [reply creating_packages/getting_started.rst]: Now is better.

**Review by lasote — APPROVED** [review]

**Review by franramirez688 — APPROVED** [review]
Looks much better and simpler 👏

File: creating_packages/getting_started.rst [hunk 4]
```diff
@@ -1,377 +1,222 @@
...
+.. important::
 
-    For this concrete example you will need, besides a C++ compiler, both *CMake* and *git*
-    installed and in your path. They are not required by Conan, so you could use your own build system
-    and version control instead.
+    This is a **tutorial** section. You are encouraged to execute these commands.
+    For this concrete example you will need **CMake** installed  in your path.
+    It is not strictly required by Conan to create packages, you can use
+    other build systems (as VS, Meson, Autotools and even your own) to create packages, without any dependency
+    to CMake.
```
Comment (czoido) [inline creating_packages/getting_started.rst:16]:
```suggestion
    For this concrete example, you will need **CMake** installed  in your path.
    It is not strictly required by Conan to create packages, you can use
    other build systems (as VS, Meson, Autotools and even your own) to do that, without any dependency
    to CMake.
```

File: creating_packages/getting_started.rst [hunk 5]
```diff
@@ -1,377 +1,222 @@
...
+The generated files are:
 
-    conanfile.py
-    test_package
-      CMakeLists.txt
-      conanfile.py
-      example.cpp
+- **conanfile.py**: On the root folder, there is a *conanfile.py* which is the main recipe file, responsible for defining how the package is built and consumed.
+- **src** folder: the *src* folder that contains the simple C++ "hello" library with a simple generic *CMakeLists.txt* to build it, with nothing specific about Conan in it.
+- (optional) **test_package** folder: contains an *example* application that links the that will require and link with the created package.
```
Comment (czoido) [inline creating_packages/getting_started.rst:38]:
```suggestion
- (optional) **test_package** folder: contains an *example* application that will require and link with the created package.
```

**Pull request merged**